docs/modules/data-links/pages/configuring-connections.adoc

```diff
@@ -1,7 +1,7 @@
+[[configuring-connections-to-external-data-stores]]
 = Configuring Connections to Data Links
 :description: You can define reusable connections to data link in your members' configuration files or with the Java member API. When you have a configured connection, you can then use it in the Pipeline API, SQL mappings, and MapStores.
```

Comment:
Shouldn't this page be deleted?

Reply:
It should. I just wanted to check something with you first. In this page to be removed, as in 5.2, we have a whole section on pre-built factories. https://docs.hazelcast.com/hazelcast/5.2/external-data-stores/external-data-stores#factories

I just want to double-check that this is no longer required and that the rework that I've done is correct.

Reply:
I think this section in the new page covers it

https://deploy-preview-685--hardcore-allen-f5257d.netlify.app/hazelcast/5.3-snapshot/data-connections/data-connections-configuration.html#connectors

However it should all say connections, not connectors